docs/Project.toml

```diff
@@ -3,9 +3,11 @@ Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 IntervalOptimisation = "c7c68f13-a4a2-5b9a-b424-07d005f8d9d2"
 SDPA = "b9a10b5b-afa4-512f-a053-bb3d8080febc"
 SumOfSquares = "4b9e565b-77fc-50a5-a571-1244f986bda1"
+TaylorModels = "314ce334-5f6e-57ae-acf6-00b6e903104a"
 
 [compat]
 Documenter = "0.24"
 IntervalOptimisation = "0.4.1"
 SDPA = "0.2, 0.3, 0.4"
 SumOfSquares = "0.3.6, 0.4, 0.5, 0.6"
+TaylorModels = "0.3, 0.4, 0.5"
```

Comment:
```suggestion
TaylorModels = "0.3, 0.4, 0.5"

```

Reply:
didn't let me commit the suggestion directly, so added it manually (I think), same for below